tests/unit/result_handlers/test_text.py

```diff
@@ -16,11 +16,13 @@
 
```

Comment:
We should add a test here to check that filter_validation_status works as intended.